css/main.css

```diff
@@ -1,11 +1,6 @@
-body {
-	padding-top: 3em;
-	max-width: 45em;
```

Comment:
Keeping a padding-top of 1em might be a good idea indeed to avoid sticking to the necessary title of introduction text above the widget.
OK for max-width.